tests/unit/dashboard/queues.spec.js

```diff
@@ -9,43 +9,47 @@ describe('dashboard instances', function() {
 		
 		$rootScope.instances = {
 			table: [{
-				spotId: '-',
-				instanceId: '-',
-				spotPrice: '-',
-				instanceType: '-',
-				spotStatus: '-',
-				instanceStatus: 'running',
-				instanceDns: '-',
-				instanceIp: '-',
-				uptime: '60',
-				tasksCompleted: '6',
-				cpuLoad: '-',
-				queueUrl: 'queue1'
+				spotId: "-",
+				instanceId: "-",
+				spotPrice: 0.04,
+				instanceType: "-",
+				spotStatus: "-",
+				instanceStatus: "running",
+				instanceDns: "-",
+				instanceIp: "-",
+				uptime: "60",
+				tasksCompleted: "6",
+				cpuLoad: "-",
+				queueUrl: "queue1"
 			}, {
-				spotId: '-',
-				instanceId: '-',
-				spotPrice: '-',
-				instanceType: '-',
-				spotStatus: '-',
-				instanceStatus: 'running',
-				instanceDns: '-',
-				instanceIp: '-',
-				uptime: '120',
-				tasksCompleted: '5',
-				cpuLoad: '-',
-				queueUrl: 'queue2'
+				spotId: "-",
+				instanceId: "-",
+				spotPrice: 0.02,
+				instanceType: "-",
+				spotStatus: "-",
+				instanceStatus: "running",
+				instanceDns: "-",
+				instanceIp: "-",
+				uptime: "120",
+				tasksCompleted: "5",
+				cpuLoad: "-",
+				queueUrl: "queue2"
 			}]
 		};
 			
-		ctrl = $controller('queuesCtrl', {$scope: $rootScope});
+		ctrl = $controller("queuesCtrl", {$scope: $rootScope});
 	}));
 	
-	it('should count workers as filtered by queueUrl', function() {
-		expect($rootScope.workerCount('queue1')).toBe(1);
+	it("should count workers as filtered by queueUrl", function() {
+		expect($rootScope.workerCount("queue1")).toBe(1);
 	});
 	
-	it('should calculate time per task', function() {
-		expect($rootScope.timePerFrame('queue1')).toBe(10);
+	it("should calculate time per task", function() {
+		expect($rootScope.timePerFrame("queue1")).toBe(10);
 	});
 	
+	it("should calculate total cost", function() {
+		expect($rootScope.totalCost("queue1")).toBe(0.04);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["expect" is not defined.](https://www.codacy.com/app/nakuljeirath/brenda-web/file/2362549761/issues/source?bid=3220248&fileBranchId=3220263#l52)